### PR TITLE
Update Lesson_1_Deploy_NFT_Bundle.md

### PR DESCRIPTION
**Pull request by TuriGuilano**
Update required node version from 12 tot 16

### CODE REVIEW

File: JS_DAO/en/Section_2/Lesson_1_Deploy_NFT_Bundle.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ We're also running this:
 
 To make sure we can retrieve the project we made using thirdweb's web app!
 
-Before executing the function, make sure you have Node 12+ installed, you can check your version with:
+Before executing the function, make sure you have Node 16+ installed, you can check your version with:
```
Comment (NickGonzalez04) [inline JS_DAO/en/Section_2/Lesson_1_Deploy_NFT_Bundle.md:120]: Hey @TuriGuilano,  just double checking things here. Did having lower than '16' throw errors?

**Comment by NickGonzalez04** [comment]
- [ ] Getting feed back on Node change

File: JS_DAO/en/Section_2/Lesson_1_Deploy_NFT_Bundle.md (hunk 1 above)
Comment (TuriGuilano) [reply JS_DAO/en/Section_2/Lesson_1_Deploy_NFT_Bundle.md:120]: Hi @NickGonzalez04 - yes my version was 12.4.0 and gave me the error. Once updated the error was gone.

**Comment by adilanchian** [comment]
@NickGonzalez04 - I think we are all good here! Node 12 is pretty darn old anyways 😅

**Comment by adilanchian** [comment]
ALSO - CONGRATS ON YOUR FIRST CONTRIBUTION 

🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖🤖

**Comment by adilanchian** [comment]
@all-contributors please add @TuriGuilano for doc

**Pull request merged**